### DISCUSSION TITLE
Some suggestions for improvements for installation wiki page

### DISCUSSION THREAD

**Discussion by Jevil7452** [Rehike/Rehike#384] [General]
1. Script which can automatically update Rehike
It's possible to have a script automatically update Rehike, here's the one I personally use:
```
@echo off
cd c:\xampp\htdocs
git pull origin master
REG ADD "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows Script Host\Settings" /v "Enabled" /t REG_DWORD /d 1 /f
cscript C:\xampp\htdocs\start.vbs
REG ADD "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows Script Host\Settings" /v "Enabled" /t REG_DWORD /d 0 /f
```
As you can see, it's a bit messy, as it uses cscript to start the OpenApache.vbs script which I moved to the Rehike folder, as well as temporarily enables VBScript (only needed for me, as I have it disabled, but this may give you some ideas on a script similiar to this
2. Move files off of Discord
Some crucial files, like certificates are being hosted on Discord at the moment. Discord is planning to make it impossible to download files from cdn.discordapp.com from external sites in the future, so i recommend moving these to another place, like this Github repo itself
3. Better way to configure yt-dlp to fix YouTube downloading
Instead of having to pass the flag every single time you want to run a yt-dlp command, you can create a file `%appdata%\yt-dlp\config.txt` with the contents `--no-check-certificates`

**Comment by ghost** [comment]
Thank you for the advice. I was unaware that we still hosted some of the files on Discord CDN, but fortunately they are just all in my private server, so reuploading them wouldn't be difficult in any case. Nor did I have any idea about the yt-dlp thing.

The documentation is in dire need of a rewrite by now, anyways. I will take these into consideration.

**Reply by ghost** [reply]
I did the last two things in the latest update to the wiki: https://github.com/Rehike/Rehike/wiki/Installation
